developer/src/kmc-package/src/compiler/package-validation.ts

```diff
@@ -41,16 +41,33 @@ export class PackageValidation {
     return true;
   }
 
-  private checkForDuplicatedLanguages(resourceType: 'keyboard'|'model', id: string, languages: KmpJsonFile.KmpJsonFileLanguage[]): void {
-    let tags: {[index:string]: boolean} = {};
+  private checkForDuplicatedOrNonMinimalLanguages(resourceType: 'keyboard'|'model', id: string, languages: KmpJsonFile.KmpJsonFileLanguage[]): boolean {
+    let minimalTags: {[tag: string]: string} = {};
+
     for(let lang of languages) {
-      const langTag = lang.id.toLowerCase();
-      if(tags[langTag]) {
-        this.callbacks.reportMessage(CompilerMessages.Warn_PackageShouldNotRepeatLanguages({resourceType:resourceType, id:id, tag:lang.id}));
-      } else {
-        tags[langTag] = true;
+      let locale;
+      try {
+        locale = new Intl.Locale(lang.id);
```

Comment:
it'll reject `en_AU` - could map underscore to hyphen if we ever wanted an error `You have en_AU but should have en-AU`

Reply:
Currently we only support kebab-case for bcp 47 in Keyman Developer; if we can keep it that way, it simplifies things for all our various parse points.